server/src/main/java/com/blackducksoftware/integration/hub/teamcity/server/global/HubGlobalServerConfigController.java

```diff
@@ -22,23 +22,12 @@
 package com.blackducksoftware.integration.hub.teamcity.server.global;
```

Comment:
I feel like THIS class makes a compelling case for the validation changes - look at all that RED!!! :)